modules/serverless-troubleshooting-knative-kafka.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/reference/troubleshooting.adoc
+
+:_content-type: REFERENCE
+[id="serverless-troubleshooting-knative-kafka_{context}"]
+= Troubleshooting Knative Kafka
+
+*How to correctly create a `KafkaChannel` or other channels?*
+
+Use the web console, the Knative CLI, or a YAML to create a channel.
+
+*Kafka Event source is not visible after installing {ServerlessOperatorName}.*
+
+Make sure to do the following:
+
+* Install the `KnativeKafka` custom resource (CR).
+* Toggle the `enabled` switch for the options you want to use to `true`.
+* Specify the bootstrap servers to use the Kafka channel or broker.
```

Comment:
This last bullet is not related to Kafka source. It's only related to Kafka channel and broker. It should be removed. IMO